portal/gil/templates/gil/base.html

```diff
@@ -163,10 +163,11 @@ <h3 class="module__head">{{_("Everyone has a part to play in improving the prost
dmiQoqVfTTHLuynq/e/+O8vvLs37amtsk5uLz79/GfwkQIECAAAEC6yYwrmXzxU6ykrs4o3/Kzz/317vv+lYNnHXrmB0RIECAAAECW1cgmct8Epc3J3E5MEXyUh5XTfrJtqM3lVhbV1LPCRAgQIAAgQ0lkGJ9T0vi8v1K8jIpobm/7dZse+6G6oyDIUCAAAECBAgUgWH4yc/kDsz7p09wyro7TqJHgAABAgQIENjQApkQ/HtJcHY3kpwsW7xgQ3fCwREgQIAAAQIElgukPs0vJbn5xoQEJ237zly+rt8JECBAgAABAjMhMAzffFAeU70tCc5/5+dH+f2tpW0mDt5BEiBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIbAiB/wfTMLF+oW4vqgAAAABJRU5ErkJggg==" fill="none" fill-rule="evenodd"/></symbol><symbol id="logo" viewBox="0 0 218 95"><title>LogoBlack</title><g fill="none" fill-rule="evenodd"><path d="M1.669 11.829h14.087c1.044 0 1.569.503 1.569 1.517v55.668c0 1.111.518 1.617 1.668 1.617h11.273c1.043 0 1.669-.506 1.669-1.617V13.346c0-1.014.524-1.517 1.565-1.517h14.091c1.042 0 1.669-.509 1.669-1.62V1.808C49.26.796 48.633.29 47.591.29H1.669C.519.29 0 .796 0 1.808v8.401c0 1.111.519 1.62 1.669 1.62M69.92 13.548c0-1.015.415-1.62 1.668-1.62h4.807c4.902 0 10.225 2.834 10.225 10.931 0 8.302-4.177 10.527-9.503 10.931h-5.424c-1.563 0-1.773-1.012-1.773-1.621V13.548zM56.975 70.631h11.379c1.044 0 1.566-.609 1.566-1.621V47.255c0-.611.21-1.725 1.876-1.825h5.638c1.251 0 1.985.708 2.297 1.618l8.245 22.366c.31.711 1.042 1.217 1.566 1.217h12.311c1.05 0 1.569-.506 1.569-1.518 0-.304-.21-.605-.313-.912L93.51 43.506c-.211-.406-.211-.81-.211-1.213.102-.506.312-1.012.625-1.316 4.07-3.241 7.622-8.808 7.622-18.726C101.546 6.869 91.105.29 76.702.29H56.975c-1.044 0-1.67.508-1.67 1.518V69.01c0 1.012.626 1.621 1.67 1.621zM131.487 72.25c14.72 0 23.172-7.085 23.172-19.433V1.808c0-1.015-.523-1.517-1.566-1.517h-11.377c-1.148 0-1.665.502-1.665 1.517v50.099c0 4
```

Comment:
Sorry, I should've been more clear. Since it doesn't have any template variables, it should be safe to move to an entirely separate resource, eg `gil/static/img/GIL.svgz`.

You should be able to create a new text file (eg `GIL.svg`) that only contains that `<svg xmlns=... >` tag, then compress it with `gzip`